docs/user-guide/recipes.rst

```diff
@@ -440,6 +440,60 @@ The Blobs and Manifest are now in the Pulp repository::
     ``skopeo copy`` looses all the tags in the repository, therefore the manifests
     need to be tagged as a separate step after uploading it.
 
+Uploading a Manifest List
+-------------------------
+
+Manifests referenced by the Manifest List must already be associated to
+the target repository. For this example, start with a synced busybox
+repository.::
+
+   $ pulp-admin docker repo sync run --repo-id busybox
+
+To upload your Manifest List, use the ``upload`` command.::
+
+   $ pulp-admin docker repo uploads upload --repo-id=busybox --manifest-list -f modified_manifest_list.json
```

Comment:
would you mind renaming to something like 'your_manifest_list.json' to get rid of the word 'modified'